legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -235,6 +235,13 @@ def delete(identifier, filing_id=None):
         if not filing:
             return jsonify({'message': _('Filing Not Found.')}), HTTPStatus.NOT_FOUND
 
+        business = Business.find_by_identifier(identifier)
+        if (filing.status == Filing.Status.DRAFT.value and
```

Comment:
This blocks them from deleting any draft filing.
Once these corp_types get added, it'll block them and behave differently than coops & BEN.
It should also check the filing type is an alteration.
It also blocks them from deleting a draft AR, CN,COD, etc.

Reply:
Good point Thor. Also, why are we preventing Alteration draft deletion in the first place? I can see this leading to a very bad ops day for someone where they have to delete everything manually a year from now. Shouldn't this always be an option for staff at least?

Reply:
ah just read your comment above about it being set to blocked